src/api/BBOPGraphUtil.js

```diff
@@ -0,0 +1,104 @@
+  return us.chain(_subjectPredicate2Objects(subjectIRI, predicate, graph))
+    .first()
+    .value();
+}
+
+export function _versionIRI2distributionIRI(versionIRI, graph) {
+  return _subjectPredicate2Object(versionIRI, 'dcat:Distribution', graph);
+}
+
+export function mergeStaticData(sourceData, staticSourceData) {
+  // iterate through sourceData and attach data from staticSourceData
+  const dynamicSummaryIRIs = [];
+  sourceData = us.map(sourceData, function fn(sourceDatum) {
+    dynamicSummaryIRIs.push(sourceDatum._summary_iri);
+    if (Object.prototype.hasOwnProperty.call(sourceDatum, '_summary_iri')) {
+      const staticDatum = staticSourceData[sourceDatum._summary_iri];
+      return Object.assign({}, sourceDatum, staticDatum);
+    }
+    return sourceDatum;
+  });
+
+  // iterate through staticSourceData and populate sources that aren't in dynamic data from biolink-api (e.g. ClinVar)
+  us.each(staticSourceData, function fn(staticDatum){
+    if(! dynamicSummaryIRIs.includes(staticDatum.summaryIRI)){
+      var newSourceEntry = populateSourceTemplate({'_summary_iri': staticDatum.summaryIRI} );
+      Object.assign(newSourceEntry, staticDatum)
+      sourceData.push(newSourceEntry);
+    }
+  });
+  return sourceData;
+}
+
+export function populateSourceFiles(sourceData, graph) {
+  for (let i = 0; i < sourceData.length; i++) {
+    const sources = _subjectPredicate2Objects(sourceData[i]._version_iri, 'dcterms:source', graph);
+    sourceData[i].sourceFiles = us.chain(sources)
+      .map(function fn(source) {
+        const node = graph.get_node(source);
+        let retVal = 'Unknown';
+        if (Object.prototype.hasOwnProperty.call(node._metadata, 'http://purl.org/pav/retrievedOn')) {
+          retVal = node._metadata['http://purl.org/pav/retrievedOn'][0];
+        }
+        return { 'fileUrl': source, 'retrievedOn': retVal };
+      })
+      .value();
+  }
+}
+
+export function populateIngestDate(sourceData, graph) {
+  for (let i = 0; i < sourceData.length; i++) {
+    sourceData[i].ingestDate =
+            graph.get_node(sourceData[i]._version_iri)._metadata['http://purl.org/dc/terms/created'][0];
+  }
+}
+
+export function populateRdfDownloadUrl(sourceData, graph) {
+  for (let i = 0; i < sourceData.length; i++) {
+    const distributionIRI = _versionIRI2distributionIRI(sourceData[i]._version_iri, graph);
+    const downloadUrls = _subjectPredicate2Object(distributionIRI, 'dcterms:downloadURL', graph);
+    sourceData[i].rdfDownloadUrl = downloadUrls.replace('MonarchArchive:', 'https://archive.monarchinitiative.org/');
+  }
+}
+
+export function populateLogoUrl(sourceData, graph) {
+  for (let i = 0; i < sourceData.length; i++) {
+    const logoUrl = _subjectPredicate2Object(sourceData[i]._summary_iri, 'schemaorg:logo', graph);
+    sourceData[i].logoUrl = logoUrl.replace(
+      'MonarchLogoRepo:', 'https://github.com/monarch-initiative/monarch-ui/blob/master/public/img/sources/'
```

Comment:
I think we can get these as static assets from our own app instead of using github, eg https://beta.monarchinitiative.org/img/sources/source-biogrid.png but you should be able to use relative paths in the vue template

Reply:
Yep, thanks Kent, you are right - fixed this